common/lib/xmodule/xmodule/templates/html/optional_reading.yaml

```diff
@@ -1,11 +1,10 @@
 ---
 metadata:
-    display_name: Optional Reading
-data: |
-      <div class="div-optionalreading">
-      <img class="icon-image" src="/static/reading.png" alt="">
-      <div>
-      <h4>Optional Reading</h4>
-      <p>Add text and links to your optional readings here.</p>
-      </div>
-      </div>
\ No newline at end of file
+  display_name: Optional Reading
```

Comment:
Nit: The display name also need to be updated to "Optional Reading or Optional Resources".

Reply:
Nice one. Sorted